mirdata/beatles.py

```diff
@@ -115,6 +191,12 @@ def load(data_home=None):
 
 
 def _load_beats(beats_path):
+    """Private function to load beat data for a track
```

Comment:
Private function to load Beatles format beat data from a file.

(Similar comment for loaders below)